### PR TITLE
Remove database usage

### LINKED ISSUE #26: Unhook database usage
The `pdp` doesn't seem to use the `unique_id` to get a file, rather it already has the filename when it comes time to make a request. As such we don't need to connect to `pcic_meta`. Since we are still early on in our development we shouldn't completely delete the database module, but rather just unhook it until further notice.

### PR DESCRIPTION
**Pull request by nikola-rados**
Demo instance is running on the usual port `30333`.

Resolves #26.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,4 +20,4 @@ RUN pip install -U pip && \
 COPY . /app
 
 EXPOSE 5000
-CMD ["gunicorn", "--bind=0.0.0.0:5000", "orca:create_app()"]
+CMD ["gunicorn", "--timeout", "600", "--bind=0.0.0.0:5000", "orca:create_app()"]
```
Comment (cairosanders) [inline Dockerfile:23]: Should we have a timeout? What about very large files?

File: requirements.txt [hunk 2]
```diff
@@ -1,7 +1,5 @@
 click==7.1.2
 dask[dataframe]
 Flask==1.1.2
-modelmeta==0.2.0
-psycopg2==2.8.6
-SQLAlchemy==1.3.23
+netCDF4==1.5.6
```
Comment (cairosanders) [inline requirements.txt:4]: Where is this used?

**Review by cairosanders — APPROVED** [review]
Looks good to me and the tests pass. I just have a couple questions.

File: Dockerfile (hunk 1 above)
Comment (nikola-rados) [reply Dockerfile:23]: I was experiencing some timeout problems with the larger files. I just wanted to add a cap for now that we can revisit later.

File: requirements.txt (hunk 2 above)
Comment (nikola-rados) [reply requirements.txt:4]: I think its a part of `xarray` or `dask`. Tests were failing without specifically adding it in.

Comment (cairosanders) [reply requirements.txt:4]: oh weird, I'm pretty sure xarray uses it but I thought it was a dependency

**Pull request merged**